### ISSUE TITLE
"plutonium fuel cell" weighs far more than it should w.r.t rm13 power armor

### ISSUE THREAD

**Issue by ConnorMartin** [CleverRaven/Cataclysm-DDA#42132]
The rm13 power armor has "plutonium fuel cells" as ammo, and has a max capacity of 5000. When loading it with 10 plutonium fuel cells, it duplicates them to 5000 fuel cells and uses them as ammo, causing a dupe glitch for the fuel cells and a massive amount of weight in the rm13 armor when fully loaded.

# Steps To Reproduce

1. Spawn rm13 power armor and plutonium cells in
2. load rm13 power armor, observe the ~1000lb weight when fully loaded, then unload it
3. note that you now have roughly 49x as many plutonium fuel cells

# Expected behavior

the armor should use some "plutonium" ammo instead of plutonium fuel cells to prevent the weight bug and the dupe glitch.

# Screenshots

![image](https://user-images.githubusercontent.com/19824381/87464848-cadcc180-c5e1-11ea-997b-83b7ed35ed5d.png)

Do not take a screenshot of the crashing window. Please provide the crash logs in the "Additional context" -->

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-3726-g8aaf68f [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Alternative Map Key [alt_map_key]
]

**Comment by hexagonrecursion** [comment]
Duplicate of #41329. Was fixed by #41599

Can not reproduce. I suggest updating to a newer version
```
- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 32 (Thirty Two); Release: 32; Codename: ThirtyTwo; 
- Game Version: 0.E-4112-g5837d3f [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]
```

**Comment by ConnorMartin** [comment]
Oh, really sorry about that. I was initially checking for dupes by searching for RM13 bugs...

I'll close the issue now. Thanks for the help.
